src/Entity/PostAddressHolderInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Entity;
+
+interface PostAddressHolderInterface
+{
+    public function getPostAddress(): ?PostAddress;
```

Comment:
what about using existing `?AddressInterface` instead of concrete class: `?PostAddress`?